ethereum/test/test_basic_inbound_channel.js

```diff
@@ -45,14 +46,16 @@ describe("BasicInboundChannel", function () {
     });
 
     it("should accept a valid commitment and dispatch messages", async function () {
-      const nonceBeforeSubmit = BigNumber(await this.channel.nonce());
+      const nonceBeforeSubmit = BigNumber(await this.channel.nonces(submitInput.params.bundle.account));
```

Comment:
Is it worth extending this test for multiple accounts? We're already using 2 accounts in the E2E test

Reply:
Leaving this to a later PR, the E2E test should be enough for now

Reply:
Sounds fine. Though see my comments further down about the E2E tests.

Reply:
Will tackle this in [SNO-306](https://linear.app/snowfork/issue/SNO-306)